crates/edr_provider/src/requests.rs

```diff
@@ -2,9 +2,12 @@
 pub mod eth;
 /// Hardhat RPC request types
 pub mod hardhat;
+mod methods;
 mod validation;
 
-pub use edr_eth::remote::{client::Request as RpcRequest, methods::MethodInvocation as EthRequest};
+pub use edr_eth::remote::client::Request as RpcRequest;
+
+pub use crate::requests::methods::{MethodInvocation, OneUsizeOrTwo, U64OrUsize};
```

Comment:
I removed the aliasing `MethodInvocation` to `EthRequest` as the public name will be `MethodInvocation` once its merged with `rpc_hardhat::Request` and the integration tests (the only external users) are using the `MethodInvocation` name.